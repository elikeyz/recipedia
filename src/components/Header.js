/* eslint-disable react/prefer-stateless-function */
import React, { Component } from 'react';
import { Link } from 'react-router-dom';
import { Navbar } from 'react-bootstrap';
import recipediaIcon from '../static/images/recipedia logo.png';

class Header extends Component {
  render() {
    return (
      <header>
        <Navbar bg="dark" variant="dark" fixed="top">
          <Link to="/">
            <Navbar.Brand>
              <img
                src={recipediaIcon}
                width="30"
                height="30"
                className="d-inline-block align-top"
                alt="Recipedia logo"
              />
              {' Recipedia'}
            </Navbar.Brand>
          </Link>
        </Navbar>
      </header>
    );
  }
}

export default Header;
