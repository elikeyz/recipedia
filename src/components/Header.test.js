import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Header from './Header';

Enzyme.configure({ adapter: new Adapter() });

describe('Header Component', () => {
  test('renders', () => {
    const wrapper = shallow(<Header />);

    expect(wrapper.exists()).toBe(true);
    expect(wrapper).toMatchSnapshot();
  });
});
