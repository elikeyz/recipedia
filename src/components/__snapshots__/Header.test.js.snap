// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component renders 1`] = `
<header>
  <Navbar
    bg="dark"
    collapseOnSelect={false}
    expand={true}
    fixed="top"
    variant="dark"
  >
    <Link
      to="/"
    >
      <NavbarBrand>
        <img
          alt="Recipedia logo"
          className="d-inline-block align-top"
          height="30"
          src="recipedia logo.png"
          width="30"
        />
         Recipedia
      </NavbarBrand>
    </Link>
  </Navbar>
</header>
`;
